.swiftlint.yml

```diff
@@ -63,3 +63,11 @@ type_name:
   max_length: # warning and error
     warning: 150
     error: 1000
+
+custom_rules:
+  ns_log_deprecation:
+    regex: "NSLog"
```

Comment:
Can we have rules to prevent the usage of print/println/os_log?

Reply:
Yes, absolutely, I just need to spend a bit of time on them to make sure I get the regex correct so it doesn't trigger in wrong places.